pkgs/development/tools/git-series/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, rustPlatform, openssl, cmake, perl, pkgconfig, zlib }:
+
+with rustPlatform;
+
+buildRustPackage rec {
+  version = "0.9.1";
+  name = "git-series-${version}";
+
+  src = fetchFromGitHub {
+    owner = "git-series";
+    repo = "git-series";
+    rev = "b5fd4aff43137a777f67771be29e7f9020c0d187";
```

Comment:
(Also, for some repositories this would need to be `rev = "v${version}";`, but not for this repository.)